wellcomeml/ml/cnn.py

```diff
@@ -224,7 +224,7 @@ def residual_attention(x1):
         elif self.feature_approach == 'sum':
             x = tf.keras.backend.sum(x, axis=1)
         elif self.feature_approach == 'concat':
-            x = tf.concat(x, axis=1)
+            x = tf.keras.layers.Flatten()(x)
```

Comment:
So you wanted one big one-dimensional vector and that one was only squashing one dimension?

Reply:
the one there was trying to concatenate two matrix on a particular axis instead of squashing so for example if you had 2 matrices of `(32,25,400)` it would create `(32,50,400)` whereas I only had one matrix to begin with and I wanted a matrix `(32,10_000)`